data/memo.go

```diff
@@ -1,11 +1,13 @@
 package data
 
+type MemoItem struct {
```

Comment:
Why the change?

Reply:
I have been doing this:

```		memos := make(data.Memos, 1)
		memos[0].Memo.MemoData = data.VariableLength(quoteData)```
Not  particularly elegant, but does your change break this code?
Irrespective of this, it's a change being slipped into a PR for another problem, so best to break it out into a separate issue/PR.
After that happy to merge.

Reply:
The proposed changes might be a breaking change depending on the way the memo is initialized. If it's initialized in the way you do - not breaching, but if in the way where you provide the embedded struct, it's breaking. 

That change is removed from that PR and will be moved to another.

Reply:
https://github.com/rubblelabs/ripple/pull/77